plugins/org.python.pydev.core/helpers/load-conda-vars.bat

```diff
@@ -10,13 +10,15 @@
 
 @set "CONDA_PREFIX=%__PYDEV_CONDA_PREFIX__%"
```

Comment:
We should change the header as it's no longer valid (we actually call the activation now, so, it's misleading).